create.sh

```diff
@@ -87,6 +87,10 @@ tools/dns.sh zone-create $RESOURCEGROUP
 tools/dns.sh a-create $RESOURCEGROUP openshift $HCPINGRESSIP
 tools/dns.sh a-create $RESOURCEGROUP openshift-tunnel $HCPINGRESSIP
 
+until $(curl --output /dev/null --silent --fail -k https://openshift.${RESOURCEGROUP}.${DNS_DOMAIN}/healthz); do
```

Comment:
If this is needed it probably indicates that the code at https://github.com/jim-minter/azure-helm/blob/master/pkg/addons/addons.go#L290 is broken and needs to be fixed

Reply:
Create a follow-up issue if appropriate.